### PR TITLE
Add 16w and 26w time range steps

### LINKED ISSUE #8652: React UI: graph does from 8w to 1y
When we click on "+" here: https://prometheus.demo.do.prometheus.io/graph?g0.expr=&g0.tab=0&g0.stacked=0&g0.range_input=8w

The duration jumps from 8 weeks to 1 year.

It looks like we could have more jumps in-between, e.g 16 weeks.

### PR DESCRIPTION
**Pull request by LeviHarrison**
I Added the `16w` and `26w` time range steps in between the `8w` and `1y` steps. I think that `8w` to `26w` is just a little bit too big of a jump, so I added in `16w` too.

Fixes #8652

### CODE REVIEW

File: web/ui/react-app/src/pages/graph/GraphControls.tsx [hunk 1]
```diff
@@ -41,6 +41,8 @@ class GraphControls extends Component<GraphControlsProps> {
     14 * 24 * 60 * 60,
     28 * 24 * 60 * 60,
     56 * 24 * 60 * 60,
+		112 * 24 * 60 * 60,
```
Comment (juliusv) [inline web/ui/react-app/src/pages/graph/GraphControls.tsx:44]: The indentation on these is tabs vs. spaces in the rest of the code (probably the linter check will yell too :)).

You can run `make react-app-lint-fix` locally by the way to fix it automatically.

Comment (LeviHarrison) [reply web/ui/react-app/src/pages/graph/GraphControls.tsx:44]: Oops, I wonder how that happened.

**Comment by LeviHarrison** [comment]
And fixed!

**Comment by juliusv** [comment]
:+1: Thanks!

**Pull request merged**